projects/sbb-esta/angular-public/autocomplete/autocomplete.md

```diff
@@ -132,6 +132,23 @@ autocomplete is attached to using the `sbbAutocompleteOrigin` directive together
 </sbb-autocomplete>
 ```
 
+### Autocomplete hint
+
+`sbb-autocomplete-hint` can be used to add additional information in option list:
+
+<!-- example(autocomplete-hint) -->
+
+```html
+<sbb-autocomplete #autoHint="sbbAutocomplete">
+  <sbb-option *ngFor="let option of filteredOptions" [value]="option">
+    {{ option.name }}
+  </sbb-option>
+  <sbb-autocomplete-hint>
+    Further results found
```

Comment:
I'd like to show here, that variables can be used in the hint
Maybe change it to `{{ remainingOptionsCount }} further results found`?